en/identity-server/next/docs/apis/organization-apis/restapis/role-management.yaml

```diff
@@ -88,6 +88,84 @@ paths:
             'https://localhost:9443/o/scim2/v2/Roles' \
             -H 'accept: application/scim+json' \
             -H 'Authorization: Bearer {bearer_token}'
+    post:
+      tags:
+        - Roles Endpoint
+      summary: Create Role
+      description: >
+        This API creates a role and returns the details of the created role
+        including its unique ID. These roles can only be associated with the 
+        applications which are created in the organization level. You cannot 
+        use these roles with the shared applications from the root organization.
+
+
+        <b>Scope(Permission) required:</b> `internal_org_role_mgt_create`
+      operationId: createRoleV2
+      requestBody:
+        content:
+          application/scim+json:
+            schema:
+              $ref: '#/components/schemas/RoleCreationObjectV2'
+      responses:
+        '201':
+          description: Valid role is created
+          content:
+            application/scim+json:
+              schema:
+                $ref: '#/components/schemas/RolePostResponseObjectV2'
+        '401':
+          description: Unauthorized
+          content:
+            application/scim+json:
+              schema:
+                $ref: '#/components/schemas/ErrorUnauthorized'
+        '403':
+          description: Forbidden
+          content:
+            application/scim+json:
+              schema:
+                $ref: '#/components/schemas/ErrorForbidden'
+        '409':
+          description: Role already exist
+          content:
+            application/scim+json:
+              schema:
+                $ref: '#/components/schemas/ErrorRoleAlreadyAvailable'
+      x-codeSamples:
+        - lang: Curl
+          source: |
+            curl -X 'POST' \
+            'https://localhost:9443/o/scim2/v2/Roles' \
+            -H 'accept: application/scim+json' \
+            -H 'Content-Type: application/scim+json' \
+            -H 'Authorization: Basic YWRtaW46YWRtaW4=' \
```

Comment:
organization API should have bearer token

Reply:
Changed